com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs

```diff
@@ -358,7 +426,7 @@ public bool TryGetScreenClearColor(ref Color color)
             return false;
         }
 
-        public IDebugDisplaySettingsPanelDisposable CreatePanel()
+        IDebugDisplaySettingsPanelDisposable IDebugDisplaySettingsData.CreatePanel()
```

Comment:
Do we not need to deprecate this function if we're changing from public to private?

Reply:
This is not a public API change in my view, because this class was internal until now. So I don't think this needs to go through deprecation.

Reply:
Ahh I see. Just wanted to be sure 👍